kratos/containers/variables_list_data_value_container.h

```diff
@@ -348,7 +347,6 @@ class KRATOS_API(KRATOS_CORE) VariablesListDataValueContainer
     const TDataType& GetValue(const Variable<TDataType>& rThisVariable, SizeType QueueIndex) const
     {
         KRATOS_ERROR_IF_NOT(mpVariablesList->Has(rThisVariable)) << "This container only can store the variables specified in its variables list. The variables list doesn't have this variable:" << rThisVariable << std::endl;    
-        KRATOS_DEBUG_ERROR_IF((QueueIndex + 1) > mQueueSize) << "Trying to access data from step " << QueueIndex << " but only " << mQueueSize << " steps are stored." << std::endl;
         return *(const TDataType*)Position(rThisVariable, QueueIndex);
```

Comment:
Here we woud check for the variable name twice (once in GetValue, once in Position)